config/initializers/01_redis.rb

```diff
@@ -12,3 +12,10 @@
   config = Rails.env.test? ? MockRedis.new : Redis.new(Redis::Config.app)
   Redis::Namespace.new('velma', redis: config, warning: true)
 end
+
+# Sherlock: Key solver
+# Used for Redis-backed locks
+$sherlock = ConnectionPool.new(size: 5, timeout: 1) do
```

Comment:
Let's not introduce a new connection pool since some of the self-hosted installations might be running on redis providers with less number of connections available.   lets reuse alfred

Reply:
Sounds good, I thought of the same, but makes sense to re-use alfred